01_lecture/homework/01_task.js

```diff
@@ -0,0 +1,24 @@
+// 1. Write the clone function so that it can clone deeply the object passed as a parameter.
+
+const cloneObj = (obj) => ({ ...obj });
```

Comment:
Its not deep clone of an object, its only a shallow copy.
What will happen when you try to copy an object with a nested object and then change that nested object?